test/TestConfig/settings.mk

```diff
@@ -58,6 +58,12 @@ else
 export JAVA_VERSION:=$(JAVA_VERSION)
 endif
 
+ifndef JVM_VERSION
```

Comment:
Please do not hard code `JVM_VERSION:=openjdk8`.  Talked with @llxia  ifndef JVM_VERSION we should construct value by JAVA_VERSION and JAVA_IMPL.
Extra code would be like following:
```
ifndef JAVA_IMPL
export JAVA_IMPL:=openj9
endif

ifeq ($(JAVA_VERSION), SE80)
	JDK_VERSION = openjdk8
else
	ifeq ($(JAVA_VERSION), SE90)
		JDK_VERSION = openjdk9
	else
		ifeq ($(JAVA_VERSION), SE100)
			JDK_VERSION = openjdk10
		endif
	endif
endif

ifneq (, $(findstring openj9, $(JAVA_IMPL)))
	JVM_VERSION = $(JDK_VERSION)-openj9
else
	JVM_VERSION = $(JDK_VERSION)
endif

```

Reply:
Can you explain why we need to "construct" JVM_VERSION using JAVA_IMPL and and JAVA_VERSION? I am just trying to understand :)) 

The way I have it in this PR seems to be working perfectly. For example, in the job below, the JVM_VERSION provided while launching the Jenkins job is 'openjdk8-openj9', and it gets passed down into the actual third party application test build script as expected:  e.g.: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/test_personal/223/console

Reply:
test_personal are using Jenkinsfile_personal, which export JVM_VERSION explicitly. JenkinsfileBase doesn't do the export JVM_VERSION explicitly. That is if you run a build with JAVA_VERSION=SE90,  JVM_VERSION will be set to openjdk8. This can be solved by export JVM_VERSION explicitly in JenkinsfileBase file.

However export JVM_VERSION is not a must. Say running locally and user set JAVA_VERSION=SE90, hard code  JVM_VERSION=openjdk8 would cause a failure.

Reply:
Thanks for the explanation. Makes sense to me now! I've made the change in the PR. 

Validated the change with personal build: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/test_personal/230/console